### PR TITLE
Improve experience of interactive GraphQL schema exploration

### PR DESCRIPTION
**Pull request by TomashKhamlai**
### Description (*)

Provide useful tips for those who explore GraphQL schema interactively.

### Fixed Issues (if relevant)

1. magento/graphql-ce#655: Add/Update docblock descriptions in GraphQL schema for modules related to Quote operations

### Manual testing scenarios (*)

1. Check that now it is clear which types were deprecated in autocomplete dropdown in GraphQL Playground

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/ConfigurableProductGraphQl/etc/schema.graphqls [hunk 1]
```diff
@@ -50,12 +50,12 @@ type AddConfigurableProductsToCartOutput {
...
 input ConfigurableProductCartItemInput {
     data: CartItemInput!
-    variant_sku: String @deprecated(reason: "Use CartItemInput.sku instead.")
+    variant_sku: String @deprecated(reason: "Use CartItemInput.sku instead.") @doc(description: "Deprecated: use CartItemInput.sku instead.")
     parent_sku: String @doc(description: "Configurable product SKU.")
     customizable_options:[CustomizableOptionInput!]
 }
 
-type ConfigurableCartItem implements CartItemInterface {
+type ConfigurableCartItem implements CartItemInterface @doc(description: "Configurable Cart Item"){
```
Comment (lenaorobei) [inline app/code/Magento/ConfigurableProductGraphQl/etc/schema.graphqls]: Please add more explanatory doc here.

File: app/code/Magento/QuoteGraphQl/etc/schema.graphqls [hunk 2]
```diff
@@ -195,7 +195,7 @@ type Cart {
     shipping_addresses: [ShippingCartAddress]! @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\ShippingAddresses")
     billing_address: BillingCartAddress! @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\BillingAddress")
     available_payment_methods: [AvailablePaymentMethod] @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\AvailablePaymentMethods") @doc(description: "Available payment methods")
-    selected_payment_method: SelectedPaymentMethod @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\SelectedPaymentMethod")
+    selected_payment_method: SelectedPaymentMethod @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\SelectedPaymentMethod") @doc(description: "Selected payment method")
```
Comment (lenaorobei) [inline app/code/Magento/QuoteGraphQl/etc/schema.graphqls]: Same here.

**Review by lenaorobei — CHANGES_REQUESTED** [review]
@TomashKhamlai looks like your PR contains redundant commits.

**Comment by TomashKhamlai** [comment]
Closed #655

**Pull request closed without merging**